Task_№8.3/Task_№8.3/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+
+namespace Task__8._3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Количество чисел");
+            int n, nomber, ost, sum;
+            n = Convert.ToInt32(Console.ReadLine());
+            sum = 0;
+            ost = 10;
```

Comment:
Присваивание `ost = 10;` лишнее. Значение 10 дальше нигде не используется.
Лучше сразу определить значение остатка и включить первое число в общую сумму. Для ввода числа можно выделить отдельный метод:
```suggestion
            nomber = GetNumber();
            sum = nomber;
            ost = nomber % 2;
```